docs/style.en-US.md

```diff
@@ -0,0 +1,180 @@
+}
+
+// temporary font size patch
+: global (.ant-tag) {
+  font-size: 12px;
+}
+```
+
+> Because antd will bring some global settings, such as font size, color, line height。 so here, you only need to focus on their own individual needs can be, without a lot of reset.
+
+### src/utils/utils.less
+
+Here you can place some tool functions for the call, such as clearing the floating `.clearfix`.
+
+### Module style
+
+Files that use for a module or page.
+
+#### Universal module level
+
+For example, `src/layouts/BasicLayout.less`, which contains some basic layout styles, is referenced by ` src/layouts/BasicLayout.js`, and the pages that use this layout do not need to care about the overall layout settings anymore. If you need to use other layouts in your project, it is also recommended to put layout-related js and less in `src/layouts`.
+
+#### Page level
+
+Specific page-related style, such as `src/routes/Dashborad/Monitor.less`, the content is only related to the content of this page. Under normal circumstances, if it is not particularly complex page, with the previous global style and common module style, there should be little to write.
+
+#### Component level
+
+This is also very simple, there are component-related styles. In your project, there are some reusing fragments in the page or relatively independent function which can be define as components, the relevant style should be extracted on the component, rather than confusing in the page.
+
+> The above style categories are for independent style files. Sometimes the style configuration is especially simple and is not repeated. You can also use the inline style `style = {{...}}`.
+
+## Override the component style
+
+Because of the special needs of the project, we often meet the need to cover the component style, here is a simple example.
+
+Antd Select In multi-select state, the default will show all the select items, here we add a limit height for display scroll bar when the content beyond this height.
+
+```js
+// TestPage.js
+import {Select} from 'antd';
+import styles from './TestPage.less'
+const Option = Select.Option;
+
+const children = [];
+for (let i = 10; i < 36; i ++) {
+  children.push(<Option key = {i.toString (36) + i}>{i.toString (36) + i}</Option>);
+}
+
+ReactDOM.render (
+  <Select
+    mode="multiple"
+    style={{width: 300}}
+    placeholder="Please select"
+    className={styles.customSelect}
+  >
+    {children}
+  </Select>
+, mountNode);
+```
+
+```css
+// TestPage.less
+.customSelect {
+  :global {
+    .ant-select-selection {
+      max-height: 51px;
+      overflow: auto;
+    }
+  }
+}
+```
+
+Two points need to note:
+
+- The imported antd component class name is not translated by CSS Modules, so the overridden class name `.ant-select-selection` must be put in `:global`.
+- Because of the previous note, the coverage is global. To prevent this from affecting other Select components, need to wrap extra className for limit the scope of the style.
```

Comment:
`To avoid affecting other Select components, the setting needs to be wrapped by an extra classname to add range restriction. `